src/features/cookies/helpers/useCookies.test.ts

```diff
@@ -209,6 +241,30 @@ describe('useCookies', () => {
         })
       })
     })
+
+    it('should not log cookies consent choice when user logs in with same account', async () => {
+      const { result } = renderHook(useCookies)
+      const { setCookiesConsent, setUserId } = result.current
+      await act(async () => {
+        await setCookiesConsent({
+          mandatory: COOKIES_BY_CATEGORY.essential,
+          accepted: ALL_OPTIONAL_COOKIES,
+          refused: [],
+        })
+      })
+      await act(async () => {
+        await setUserId(FAKE_USER_ID)
+      })
+
+      await act(async () => {
+        await setUserId(FAKE_USER_ID)
+      })
+
+      const setCookieConsent = 1
+      const setUserIdAfterlogin = 1
+      const apiCalledTimes = setCookieConsent + setUserIdAfterlogin
```

Comment:
Pour ne pas les confondre avec les fonctions peut-être ?
```suggestion
      const SET_COOKIE_CONSENT = 1
      const SET_USER_ID_AFTER_LOGIN = 1
      const apiCalledTimes = SET_COOKIE_CONSENT + SET_USER_ID_AFTER_LOGIN
```